src/components/ExpenseForm/ExpenseForm.jsx

```diff
@@ -1,23 +1,49 @@
 import React, { useState, useEffect } from 'react'
 import { DataService } from '../../services/DataService'
 import DatePicker from 'react-datepicker'
-import { expensesUrl } from '../../static/apiconfig'
+import { EXPENSES_URL } from '../../static/apiconfig'
+import ChooseCategory from '../ChooseCategory/ChooseCategory'
+import ChooseSubcategory from '../ChooseSubcategory/ChooseSubcategory'
+import { CATEGORIES_URL } from '../../static/apiconfig'
 
 import 'react-datepicker/dist/react-datepicker.css'
 
 import * as Styled from './style'
 
-const  ExpenseForm = () => {
+const ExpenseForm = () => {
   const [cost, setCost] = useState('')
-  const [category, setCategory] = useState('')
+  const [categories, setCategories] = useState()
+  const [categoryName, setCategoryName] = useState()
+  const [subcategory, setSubcategory] = useState('')
+  const [subCategories, setSubCategories] = useState('')
   const [date, setDate] = useState(new Date())
   const [isSubmitting, setSubmitting] = useState(false)
+  const categoryType = 'EXPENSES'
+  
+  useEffect(() => {
+    (async function() {
+      try {
+        await fetch(`${CATEGORIES_URL}?type=${categoryType}`)
```

Comment:
```
const response = await fetch(`${CATEGORIES_URL}?type=${categoryType}`)
setCategories(response.json())
```
Maybe this way?

Reply:
I change it for that:
const response = await fetch(`${CATEGORIES_URL}?type=${categoryType}`)
        const data = await response.json();
        setCategories(data)